domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/startlesson/Types.kt

```diff
@@ -12,10 +13,6 @@ data class SecondName(val name: NonEmptyText)
 
 data class ClassName(val name: NonEmptyText)
 
```

Comment:
what do you think of separating student types between class registry context and attendance checking context? 
From business context I see it that way:
1. when we talk of class registry, there is a Student record and it has name and number on the list (so maybe the Student in the context of class registry could be renamed to StudentRecord)
2. when we check the attendance, we take the student record, and we mark student presence (and this student is something different from the record, because he's a real person)

Reply:
That makes a lot of sense. I will correct that.